dissect/target/plugins/os/unix/generic.py

```diff
@@ -20,6 +22,41 @@ def activity(self):
                 last_seen = f.stat().st_mtime
 
         if last_seen != 0:
-            return datetime.datetime.fromtimestamp(last_seen)
+            return datetime.fromtimestamp(last_seen)
 
-        return
+        return "unknown"
```

Comment:
I understand why this can be useful, but from an API perspective I think it makes more sense to keep returning `None` here. Same for the install date. This intention could however be reflected in `target-info` by printing `Unknown` for the textual representation.